src/Kafka/Settings/Settings.tsx

```diff
@@ -0,0 +1,148 @@
+import {
+  Alert,
+  AlertActionCloseButton,
+  AlertGroup,
+  AlertVariant,
+  Button,
+  Card,
+  CardBody,
+  Flex,
+  FlexItem,
+  Level,
+  LevelItem,
+  Modal,
+  ModalVariant,
+  Page,
+  PageSection,
+  Spinner,
+  Switch,
+  Title,
+} from "@patternfly/react-core";
+import type { AlertProps } from "@rhoas/app-services-ui-shared";
+import type { FunctionComponent } from "react";
+import { useTranslation } from "react-i18next";
+
+export type SettingsProps = {
+  showAlert: AlertProps[];
+  isChecked: boolean;
+  isModalOpen: boolean;
+  onChange: () => void;
+  onDisable: () => void;
+  onClose: () => void;
+  closeAlertAction: (key: string | undefined) => void;
+  isLoading: boolean;
+};
+
+export const Settings: FunctionComponent<SettingsProps> = ({
+  showAlert,
+  isChecked,
+  onChange,
+  isModalOpen,
+  onDisable,
+  onClose,
+  closeAlertAction,
+  isLoading,
+}) => {
+  const { t } = useTranslation("kafka");
+
+  return (
+    <>
+      <PageSection>
+        <Card>
+          <CardBody>{t("settings.kafka_instance_settings_label")}</CardBody>
+        </Card>
+      </PageSection>
+      <Page>
+        <PageSection isWidthLimited isCenterAligned>
+          <Card>
+            <CardBody>
+              <Level>
+                <LevelItem>
+                  <Flex>
+                    <FlexItem spacer={{ default: "spacerNone" }}>
+                      <Title headingLevel={"h3"}>
+                        {t("settings.connection_re_authentication_label")}
+                      </Title>
+                    </FlexItem>
+                    <FlexItem>
+                      {isChecked ? (
+                        "On"
+                      ) : isLoading ? (
+                        <>
+                          <Spinner
+                            size="md"
+                            aria-valuetext={"Turning off..."}
+                            aria-label={"Turning off..."}
```

Comment:
A spinner is a component that provides status. The `aria-valuetext` is the attribute that communicates what the status is, but the `aria-label` should not. Instead, `aria-label` should describe what we're providing status for, which is the setting. 

Therefore, the value for `aria-label` here should reuse the label of the setting, e.g. `connection_re_authentication_label`